### PR TITLE
Issue #394: created CHANGELOG.adoc on master

### LINKED ISSUE #394: Track addressed issues
Use some mechanic like Milestones to track issues addressed within a given time period (e. g. a sprint), and produce a changelog for each such period.

### PR DESCRIPTION
**Pull request by inoxx03**
@tradej @zrhoads WDYT?

* Formatted as `.adoc` instead of  `.md`
* Messages have been edited to provide more context for each issue.
* The formatting is intentionally simple. Is this sufficiently detailed? Do we need to include any other info?
* If this gets approved and merged I'll file a PR to include a similar log (sans the _Current Development_ section) for the `production` branch, too.

resolves #394

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+= Change Log
+
+== Current Development]
```
Comment (tradej) [inline CHANGELOG.adoc]: Extra `]`.

Comment (inoxx03) [inline CHANGELOG.adoc]: ah, Thanks, must've missed that one. Will fix momentarily.

**Comment by inoxx03** [comment]
thanks @tradej, typo has been removed.

**Comment by rhoads-zach** [comment]
merged: 62d847f2f3daadacc217219a783fbdd8d0422223

**Pull request closed without merging**